etc/vlc.profile

```diff
@@ -25,5 +25,6 @@ private-bin vlc,cvlc,nvlc,rvlc,qvlc,svlc
 private-dev
 private-tmp
 
+# memory-deny-write-execute
```

Comment:
Sure! Should I also put a warning note about mdwe into the cvlc profile? I can't test it on my current machine.